lib/services/lambda/index.js

```diff
@@ -163,6 +163,9 @@ exports.consumeEvents = function (ownServiceContext, ownDeployContext, producerS
             principal = producerDeployContext.eventOutputs.principal;
             sourceArn = producerDeployContext.eventOutputs.eventRuleArn;
         }
+        else if (producerServiceType === 'alexaskillkit') {
+            principal = 'alexa-appkit.amazon.com';
```

Comment:
I think you'll need to provide a sourceArn here as well, but I'm not sure what that would be for the Alexa Skill Kit. It's been required on every other service I've done though.